### PR TITLE
issue: 1081001 Add cyclic buffer readable status

### PR DESCRIPTION
**Pull request by serglelv**
Returns the size of the packet arrived to MP_RQ cyclic buffer and can be extended to provide similar to the poll functionality. Please review.
@rafiw @ilansmith  
Signed-off-by: Sergey Lykov <sergeyly@mellanox.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1553/ for details (Mellanox internal link).

File: src/vma/dev/ring_eth_cb.cpp [hunk 1]
```diff
@@ -225,6 +225,19 @@ inline bool ring_eth_cb::reload_wq()
...
+{
+	volatile struct mlx5_cqe64 *cqe64;
+	uint16_t size = 0;
+	uint32_t flags = 0;
+
+	int ret = ((cq_mgr_mp *)m_p_cq_mgr_rx)->poll_mp_cq(size, m_curr_wqe_used_strides, flags, cqe64);
+	if (unlikely(ret == -1)) {
+		return ret;
+	}
+	return size;
```
Comment (rafiw) [inline src/vma/dev/ring_eth_cb.cpp]: this returns the size of the packet arrived

Comment (serglelv) [inline src/vma/dev/ring_eth_cb.cpp]: Corrected. Thanks.

Comment (serglelv) [inline src/vma/dev/ring_eth_cb.cpp]: The vma_cyclic_buffer_is_readable is not needed in VMA and the functionality will be moved to the adaptation layer.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1559/ for details (Mellanox internal link).

**Pull request closed without merging**